src/applications/personalization/profile/tests/routes.unit.spec.js

```diff
@@ -0,0 +1,28 @@
+import { expect } from 'chai';
+
+import getRoutes from 'applications/personalization/profile/routes.js';
+import { PROFILE_PATH_NAMES } from 'applications/personalization/profile/constants.js';
+
+describe('getRoutes', () => {
+  describe('when options.removeDirectDeposit is false', () => {
+    it('should return all possible PROFILE_PATHS`', () => {
```

Comment:
nit: this doesn't quite accurately reflect what's being tested. can it be changed to `should return the direct deposit path`